crates/scroll/primitives/src/receipt.rs

```diff
@@ -2,9 +2,12 @@ use alloy_consensus::{
     proofs::ordered_trie_root_with_encoder, Eip2718EncodableReceipt, Eip658Value, Receipt,
```

Comment:
I think it's worth to review this file, the rest is mostly moving functions around.